app/src/renderer/components/common/TmToolBar.vue

```diff
@@ -0,0 +1,31 @@
+<template lang="pug">
+  tm-tool-bar
+    slot
+    a.back(@click="$router.go(-1)" v-tooltip.bottom="'Back'")
+      i.material-icons arrow_back
+    a.help(@click="enableModalHelp" v-tooltip.bottom="'Help'")
+      i.material-icons help_outline
+    a.sign-out(@click="signOut" v-tooltip.bottom.end="'Sign Out'")
+      i.material-icons exit_to_app
+</template>
+
+<script>
+import { TmToolBar } from "@tendermint/ui"
+export default {
+  name: "vm-tool-bar",
+  components: {
+    TmToolBar
+  },
+  mounted() {
+    // console.log(TmToolBar)
```

Comment:
?

Reply:
This used to be called VrToolBar and it extended @tendermint/ui/TmToolBar because all voyager pages have those three slots in common. After the last review from @peng he suggested we name all the components Tm but made the names different withiin the component (tm-tool-bar versus vm-tool-bar) so that they would render without recursion.